android-components/components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml

```diff
@@ -120,7 +120,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:textSize="14sp"
-                    android:textColor="@color/photonRed70"
+                    android:textColor="@color/mozac_feature_addons_error_text_color"
```

Comment:
This is needed to have a different red color in dark vs. light mode.

Reply:
👍🏽

Reply:
Initially I assumed that the `photonRed70`will look good in both themes, thanks for addressing it!

Reply:
Same but I asked UX again about it.